packages/balance/src/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import singleSpaReact from 'single-spa-react'
+import 'normalize.css'
+
+import App from './App'
+
+const lifecycles = singleSpaReact({
+  // eslint-disable-next-line no-unused-vars
+  errorBoundary (err, info, props) {
```

Comment:
```suggestion
  errorBoundary (err) {
```

se só não passar as props não seria o suficiente pra não precisar do disable do lint?

Reply:
Talvez seja interessante manter todos os argumentos, pra gente lembrar que eles existem. Já que isso pode ser útil quando for fazer a issue específica de tratamento de erros. Ai caso mais pra frente não seja usado, a gente remove.